src/e

```diff
@@ -90,6 +92,32 @@ program
     'Loads required versions of Xcode and the macOS SDK and symlinks them.  This may require sudo',
   );
 
+program
+  .command('depot-tools')
+  .alias('d')
+  .description('Run a command from the depot-tools directory with the correct configuration')
+  .allowUnknownOption()
+  .action(() => {
+    const args = process.argv.slice(3);
+    let cwd;
+    if (args[0] === 'goma_ctl' || args[1] === 'goma_auth') {
+      cwd = goma.dir(evmConfig.current().root);
+      args[0] = `${args[0]}.py`;
+      args.unshift('python');
+    }
+
+    const { status, error } = depot.spawnSync(evmConfig.current(), args[0], args.slice(1), {
+      cwd,
```

Comment:
(Is it safe to pass an undefined `cwd` to spawnSyncs' options?)

Reply:
Yes 👍